doc/api/code_integrity.md

```diff
@@ -0,0 +1,116 @@
+# Code Integrity
+
```

Comment:
We should add the stability to 1.1 Active Development

Reply:
Do you have a preferred way of resolving comments? I normally just let people know that I addressed their comment and let them resolve it themselves, or I can resolve it for you if it's easier.

Reply:
You can resolve it. No problem.